packages/wonder-blocks-form/src/components/__tests__/text-area.test.tsx

```diff
@@ -988,10 +989,48 @@ describe("TextArea", () => {
                     />,
                     defaultOptions,
                 );
+
+                // Assert
+                expect(validate.mock.calls).toStrictEqual([["text"], ["text"]]);
+            });
+
+            it("should call the onValidate function only once when it is first rendered (once after mount)", async () => {
+                // Arrange
                 // Act
+                const onValidate = jest.fn();
+                const errorMessage = "Error message";
+                render(
+                    <TextArea
+                        value="text"
+                        onChange={() => {}}
+                        validate={() => errorMessage}
+                        onValidate={onValidate}
+                    />,
+                    defaultOptions,
+                );
 
                 // Assert
-                expect(validate).toHaveBeenCalledExactlyOnceWith("text");
+                expect(onValidate).toHaveBeenCalledExactlyOnceWith(
```

Comment:
**praise:** TIL about `toHaveBeenCalledExactlyOnceWith`! I previously used `toHaveBeenCalledOnce` but it was removed from the API some time ago, so it's great to see that we can still use something like that :).